protos/shared/shared.proto

```diff
@@ -7,10 +7,19 @@ option go_package = "shared";
 import "github.com/gogo/protobuf/gogoproto/gogo.proto";
 import "google/protobuf/timestamp.proto";
 
+service RuntimeService {
+  rpc SharedCheckRuntime(SignedRuntimeInfoRequest) returns (RuntimeInfoReport);
```

Comment:
can use the name CheckRuntime() since this will be the only CheckRuntime() we are going to use 😃

Reply:
ok